demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const localAttendeeId = (audioVideo as any).audioVideoController
```

Comment:
Can we get the `attendeeId` from the `meetingManager.configuration...`? Not sure, if we have used `realtimeController` for local attendee in React SDK or JS SDK, but have seen we using it from meeting session configuration.

Reply:
Sure. I will change it. I just followed the example here in useAttendeeStatus to use the `realtimeController`: https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/hooks/sdk/useAttendeeStatus.tsx#L26

Reply:
Resolving since this is updated now.